### ISSUE TITLE
Add parameter ignoreSex to `exclusionMatrix()`

### ISSUE THREAD

**Issue by thoree** [magnusdv/dvir#65]
``` r
# Could exclusionMatrix() gain an argument, ignoreSex,
# and return NA in the first row of the exclusion matrix if ignoreSex = F (default):
library(dvir)
#> Loading required package: pedtools
example1$pm["V1"] = swapSex(example1$pm["V1"], "V1")
example1
#> DVI dataset: 
#>  3 victims (2M/1F): V1, V2, V3
#>  3 missing (3M/0F): M1, M2, M3
#>  1 typed refs: R1
#>  1 ref family: (unnamed)
#> Number of markers, PM and AM: 1
exclusionMatrix(example1)
#>    M1 M2 M3
#> V1  0  0  0
#> V2  0  1  0
#> V3  0  0  0
```

<sup>Created on 2023-10-13 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

**Comment by magnusdv** [comment]
This is already done in the exclusion matrix reported by `findExcluded()`:

``` r
library(dvir, quietly = T)

example1$pm = swapSex(example1$pm, "V1")

e = findExcluded(example1, verbose = F)
e$exclusionMatrix
#>    M1 M2 M3
#> V1 NA NA NA
#> V2  0  1  0
#> V3  0  0  0
```
Is this the expected output?

The `exclusionMatrix()` was a precursor to `findExcluded()`, and I had forgotten about it. I'll either remove it or syncronize them.

**Comment by magnusdv** [comment]
Closing this as fixed in version 3.2.1.
